site/pages/[...slug].tsx

```diff
@@ -45,31 +44,42 @@ export default function Page({ source, meta, sidebarTree }) {
 
   return (
     <>
-      <NextSeo
-        title={title}
-        description={description}
-        canonical={canonicalUrl}
-        openGraph={{
-          url: canonicalUrl,
-          title,
-          description,
-          type: meta.layout === 'blog' ? 'article' : 'website',
-          article: meta.layout === 'blog' && {
-            publishedTime: meta.date,
-            authors: meta.authorsDetails?.map(a => a.url) || [],
-          },
-          images: [
-            {
-              url: image,
-              alt: title,
+      {meta.layout === 'docs' ?
```

Comment:
@Datopian-Shreyas something seems wrong here, are we saying that following NextSeo only applies when layout is docs? but then we have `meta.layout === 'blog'` check which never true.